test/unit/login.js

```diff
@@ -49,8 +49,7 @@ describe('Login', () => {
         Login._read.onFirstCall().rejects(new Error('robot'))
         return assert.isRejected(
           Login.login(mockArgs),
-          Error,
-          /robot/
```

Comment:
I think having this message check here is still correct. This prevents something like "cannot call _something_ of undefined" from coming back as the rejection error and the test still passing.